api_register.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
Can you please bring this code into api_profile.go rather than a new file, since this has to do with user profile management (e.g. creating a new profile)?